internal/suggestions/client.go

```diff
@@ -33,10 +33,7 @@ type suggestionRequest struct {
 }
 
 func Upload(schema []byte, filePath string, model string) (string, string, error) {
-	openAIKey, err := GetOpenAIKey()
-	if err != nil {
```

Comment:
Think I'd rather keep this check, since error-ing here is clearer than just not setting `x-openai-key` header if key is set. Seems like `OPENAI_API_KEY` is going to be set at all times in the env, but from the context of this method, this check still makes sense to me.